aws_lambda_powertools/utilities/idempotency/idempotency.py

```diff
@@ -66,7 +67,9 @@ def idempotent(
         >>>     return {"StatusCode": 200}
     """
 
-    if os.getenv(constants.IDEMPOTENCY_DISABLED_ENV):
+    # Check if the IDEMPOTENCY_DISABLED_ENV environment variable is set to True
+    # If IDEMPOTENCY_DISABLED_ENV is not set or set to any false value, the function will be idempotent by default
+    if strtobool(os.getenv(constants.IDEMPOTENCY_DISABLED_ENV, "false")):
         return handler(event, context, **kwargs)
```

Comment:
let's add a warning as it's not something customers would naturally want it disabled in production. It's easy to set and forget as we've learned from Metrics and other features.

Reply:
Hey @heitorlessa! I have created a new class, PowertoolsWarning, which inherits from UserWarning. This new class enables customers to filter and handle specific warnings raised by Powertools.

Reply:
niiice! need to think of a good naming so the intent is clear. I'll come back today